src/pages/api/webhooks/saleor/product-updated.ts

```diff
@@ -0,0 +1,68 @@
+import { NextWebhookApiHandler, SaleorWebhook } from "@saleor/app-sdk/handlers/next";
+import { gql } from "urql";
+import { ProductUpdatedWebhookPayloadFragment } from "../../../../../generated/graphql";
+import { saleorApp } from "../../../../../saleor-app";
+
+// Next.js body parser has to be turned off for us to be able to access the raw request body
+// which is required to validate incoming requests
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+// By using the fragment, we'll be able to get TS types for the payload
+// Since this webhook subscribes for product updates, we have to create fragment on
+// type ProductUpdated
+export const ProductUpdatedWebhookPayload = gql`
+  fragment ProductUpdatedWebhookPayload on ProductUpdated {
+    product {
+      id
+      name
+    }
+  }
+`;
+
+// Subscription query which will be used to construct response for our webhook
+export const ExampleProductUpdatedSubscription = gql`
+  ${ProductUpdatedWebhookPayload}
+  subscription ExampleProductUpdated {
+    event {
+      ...ProductUpdatedWebhookPayload
+    }
+  }
+`;
+
+// The SaleorWebhook class will help us define the webhook and provide handlers for API and manifest
+// ProductUpdatedWebhookPayloadFragment is a fragment object generated by the codegen. We are using it to
+// provide webhook payload types
+export const productUpdatedWebhook = new SaleorWebhook<ProductUpdatedWebhookPayloadFragment>({
+  name: "Example product updated webhook",
+  webhookUrl: `/api/webhooks/saleor/product-updated`,
```

Comment:
change to `webhookPath`

Reply:
Check if we can do this scenario:
1. User has wrong webhookPath (eg copy/pased file)
2. Webhook doesn't work
3. SDK/SaleorWebhook detects error, compares URL from request and webhookPath and *suggests* that it could be an issue

Reply:
In case of copy/paste:
Event will be rejected due to event mismatch (don't think there will be many apps with multiple webhooks for the same event).

I'm writing docs with debugging instructions for situations like this